sci-libs/atlas/atlas-3.10.3.recipe

```diff
@@ -0,0 +1,138 @@
+SUMMARY="Automatically Tuned Linear Algebra Software"
+DESCRIPTION="A software library for linear algebra. It provides a mature open \
+source implementation of BLAS APIs for C and Fortran77."
+HOMEPAGE="http://math-atlas.sourceforge.net/"
+COPYRIGHT="2011-2016 The ATLAS Project"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://downloads.sf.net/math-atlas/atlas3.10.3.tar.bz2"
+CHECKSUM_SHA256="2688eb733a6c5f78a18ef32144039adcd62fabce66f2eb51dd59dde806a6d2b7"
+SOURCE_DIR="ATLAS"
+PATCHES="atlas-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+
+PROVIDES="
+	atlas$secondaryArchSuffix = $portVersion
+	lib:libatlas$secondaryArchSuffix = $portVersion
+	lib:libcblas$secondaryArchSuffix = $portVersion
+	lib:libf77blas$secondaryArchSuffix = $portVersion
+	lib:liblapack$secondaryArchSuffix = $portVersion
+	lib:libptcblas$secondaryArchSuffix = $portVersion
+	lib:libptf77blas$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libgfortran$secondaryArchSuffix
+	lib:libgomp$secondaryArchSuffix
+	"
+CONFLICTS="
+	blas$secondaryArchSuffix
+	cblas$secondaryArchSuffix
+	lapack$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	atlas${secondaryArchSuffix}_devel = $portVersion
+	devel:libatlas$secondaryArchSuffix = $portVersion
+	devel:libcblas$secondaryArchSuffix = $portVersion
+	devel:libf77blas$secondaryArchSuffix = $portVersion
+	devel:liblapack$secondaryArchSuffix = $portVersion
+	devel:libptcblas$secondaryArchSuffix = $portVersion
+	devel:libptf77blas$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	atlas$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	binutils$secondaryArchSuffix
+	cmd:awk
+	cmd:bzip2
+	cmd:gfortran$secondaryArchSuffix
+	cmd:fgrep
+	cmd:find
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:sed
+	cmd:pwd
```

Comment:
You don't need **`cmd:pwd`** in BUILD_PREREQUIRES since it is provided by coreutils.